Terminal.Gui/Configuration/Scope.cs

```diff
@@ -3,209 +3,74 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using System.Text.Json;
-using System.Text.Json.Serialization;
 using static Terminal.Gui.ConfigurationManager;
 
 #nullable enable
 
 namespace Terminal.Gui {
```

Comment:
While you're in  here, can you please change this to use `namespace Terminal.Gui;`?

Reply:
I've changed all the new files to use file scoped namespace statements 🚀